### PR TITLE
feat(sharing): Change message for shared links for Canvas files

### PR DESCRIPTION
**Pull request by pjzabinski**

### CODE REVIEW

File: src/features/unified-share-modal/SharedLinkAccessDescription.js [hunk 1]
```diff
@@ -3,23 +3,31 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
 import type { ItemType } from '../../common/types/core';
+import { isBoxCanvas } from '../../utils/file';
+import convertToBoxItem from './utils/item';
 
 import { ANYONE_WITH_LINK, ANYONE_IN_COMPANY, PEOPLE_IN_ITEM } from './constants';
-import type { accessLevelType } from './flowTypes';
+import type { accessLevelType, item as itemtype } from './flowTypes';
```
Comment (greg-in-a-box) [inline src/features/unified-share-modal/SharedLinkAccessDescription.js:10]: were you unable to just import `item`as is, there are other types that do not have the word "Type" at the end and they are imported as is?

Comment (pjzabinski) [reply src/features/unified-share-modal/SharedLinkAccessDescription.js:10]: The thing is that there's either an argument or a constant called `item` that is coming from `Props`, so there could potentially be a conflict with those 🤔 Also, I followed what was already done in SharedLinkSection.js. Should we try and just use `item` as is here?

Comment (tjuanitas) [reply src/features/unified-share-modal/SharedLinkAccessDescription.js:10]: yeah looks to be done that way for naming conflicts. seems okay to keep

**Review by tjuanitas — DISMISSED** [review]
lgtm. minor question

File: src/features/unified-share-modal/__tests__/SharedLinkAccessDescription.test.js [hunk 2]
```diff
@@ -8,16 +8,35 @@ describe('features/unified-share-modal/SharedLinkAccessDescription', () => {
...
                 itemType: 'folder',
+                item: {
+                    extension: '',
+                },
             },
-        ].forEach(({ itemType }) => {
+            {
+                itemType: 'folder',
+                item: {
+                    extension: 'boxcanvas',
```
[the remark this answers is not in the document]
Comment (pjzabinski) [reply src/features/unified-share-modal/__tests__/SharedLinkAccessDescription.test.js]: Right, removed to avoid confusion.

**Review by tjuanitas — APPROVED** [review]

**Review by tjuanitas — APPROVED** [review]

**Review by greg-in-a-box — APPROVED** [review]

**Pull request merged**